src/objects/Allocations_Settings__c.object

```diff
@@ -49,6 +49,14 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Payment_Allocation_Two_Way_Sync_Enabled__c</fullName>
```

Comment:
I assume @lianeyee and @markross-sfdo are still working on the description and help text for this, or are we forgoing them for this field?

Reply:
Just added the description and help text.